campaigns/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.urls import path
+
+from .views import (
+    CampaignIdView,
+    CampaignView,
+    DonationView,
+    # EndCampaignView,
```

Comment:
Are you planning to implement these or can we delete those comments?

Reply:
Will be the donation endpoint import…